### ISSUE TITLE
Sex specific comps in writeComps

### ISSUE THREAD

**Issue by chantelwetzel-noaa** [nwfsc-assess/PacFIN.Utilities#6]
Need to add if statement at the end of the function based on the value specified for "out" in the function input (Mout, Fout, FthenM).  Currently, it writes all options where the FthenM option is the final output that the user gets.

**Comment by andi-stephens** [comment]
That was by design, actually, the idea being that if you later decided you
wanted the comps in a different format, you'd have all of them handy.

The "out" argument is for debugging purposes (mine, for debugging code,
yours, for previewing data).

--A.

**Comment by CWetzel** [comment]
This is deceptive to the user because they can specify Mout or Fout but
they still receive FthenM.  Removing the function input would clarify to
the user that they will always receive FthenM.

**Comment by andi-stephens** [comment]
I agree.  I think Kelli put that "out" argument in when familiarizing
herself with the code.

**Comment by taylori** [comment]
The problem discussed in this issue has been addressed in recent commit a7a299d5575f0dcfa341b11b93c359e6ecf6ef01.

As part of that change, the "out" argument has been renamed "returns".

**Comment by iantaylor-NOAA** [comment]
It turns out that this bug wasn't quite fixed in 2017 after all. Lines 465-466 of the commit referenced above are missing an `append = TRUE` so the first 3 tables in the file have always been overwritten when the 4th table was written:
```
  IDstring = paste("\n\n", "Females then males")
  cat(file=fname, IDstring, "\n")
```

This is fixed in commit 37150c9a3e204711c555c623aca8fe68931e39d3 of the `ian_suggestions` branch, but should probably be fixed in master if it's not going to mess up anyone's workflow.
